lib/papertrail/cli.rb

```diff
@@ -137,12 +142,31 @@ def query_time_range
       end
     end
 
+    COLORS = [:cyan, :yellow, :green, :magenta, :red]
+
+    def colorize(event)
+      idx = (event.data["hostname"].length + event.data["program"].length) % 5
```

Comment:
> What about combining these with another gap you identified, and changing `--force-colors` into `--colors={h,p,hp,none}`? The default would be `--colors=hp` (your 2nd example) when color terminal emulation was detected, but someone could also use `--colors=p` for the same behavior as Heroku's CLI (and your 1st example) or `--colors=h` to make it easier to spot multiline output from the same sender.

I could see that being useful. I think that's worth exploring in a new PR. No sense holding up colors while we discuss if/how to accomplish it.